pkg/autoscaler/k8sclient/k8sclient.go

```diff
@@ -140,17 +162,31 @@ type ClusterStatus struct {
 }
 
 func (k *k8sClient) GetClusterStatus() (clusterStatus *ClusterStatus, err error) {
-	opt := metav1.ListOptions{Watch: false}
-
-	nodes, err := k.clientset.CoreV1().Nodes().List(opt)
-	if err != nil || nodes == nil {
+	// TODO: Consider moving this to NewK8sClient method and failing fast when
+	// reflector can't initialize. That is a tradeoff between silently non-working
+	// component and explicit restarts of it. In majority of the cases the restart
+	// won't repair it - it may just give better visibility into problems.
```

Comment:
Maybe "it may just give" -> "but it may give"?

Reply:
Fair enough - done.